CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 2017-08-18 - Version 4.1.0
+
+### Summary
+
+This is a minor release, with bug fixes and documentation changes
+
+### Added
+
+- Added ability to update physical path and application pool for sites ([MODULES-5125](https://tickets.puppetlabs.com/browse/MODULES-5125))
+- Added testing of module on Puppet 5 ([MODULES-5187](https://tickets.puppetlabs.com/browse/MODULES-5187))
+- Added more acceptance testing of Application Pool settings ([MODULES-5195](https://tickets.puppetlabs.com/browse/MODULES-5195))
+- Added iis_virtual_directory to README ([MODULES-5433](https://tickets.puppetlabs.com/browse/MODULES-5433))
```

Comment:
Perhaps just opinion, but I would generally be okay putting docs/example/test changes into "Fixed" and just doing a bugfix release for those kinds of things. Updating application pool paths for sites is definitely a feature though.